### DISCUSSION TITLE
any plans to update to Svelte 5?

### DISCUSSION THREAD

**Discussion by lit-carlos** [urania-dev/snapp#69] [General]
any plans to update to Svelte 5?

**Comment by urania-dev** [comment]
TBF this app was born as a way to learn svelte5 towards release candidate. It Is already in svelte5 🧐

**Comment by lit-carlos** [comment]
Oh, excuse the ignorance.

**Reply by urania-dev** [reply]
don't worry, it is a very valid question, migration won't take long from beta to rc :) next update will probably be
